### PR TITLE
Add allowedCloudEndpoints property to crossTenantAccessPolicy

### PR DESCRIPTION
**Pull request by jkdouglas**

### CODE REVIEW

File: changelog/Microsoft.DirectoryServices.json [hunk 1]
```diff
@@ -1,5 +1,23 @@
...
+        {
+          "Id": "13a51673-7133-48f3-bdfe-e6a8309de690",
+          "ApiChange": "Resource",
+          "ChangedApiName": "crossTenantAccessPolicy",
+          "ChangeType": "Addition",
+          "Description": "Added the `allowedCloudEndpoints` property to the [crossTenantAccessPolicy](https://docs.microsoft.com/en-us/graph/api/resources/crosstenantaccesspolicy?view=graph-rest-beta) resource.",
+          "Target": "crossTenantAccessPolicy"
+        }
+      ],
+      "Id": "34b0a990-a3c9-4b70-92ac-a8ac93bf4860",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.DirectoryServices.json:14]: - Aligning the values with the [guidance](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211742/Changelog)
- Same GUID for all objects in the changelog record

```suggestion
          "ApiChange": "Property",
          "ChangedApiName": "allowedCloudEndpoints",
          "ChangeType": "Addition",
          "Description": "Added the **allowedCloudEndpoints** property to the [crossTenantAccessPolicy](https://docs.microsoft.com/en-us/graph/api/resources/crosstenantaccesspolicy?view=graph-rest-beta) resource.",
          "Target": "crossTenantAccessPolicy"
        }
      ],
      "Id": "13a51673-7133-48f3-bdfe-e6a8309de690",
```

**Review by FaithOmbongi — APPROVED** [review]
Thank you @jkdouglas for your work on this PR. Approved with a minor change requested.

File: changelog/Microsoft.DirectoryServices.json [hunk 2]
```diff
@@ -1,5 +1,23 @@
 {
   "changelog": [
+    {
+      "ChangeList": [
+        {
+          "Id": "13a51673-7133-48f3-bdfe-e6a8309de690",
+          "ApiChange": "Property",
+          "ChangedApiName": "allowedCloudEndpoints",
+          "ChangeType": "Addition",
+          "Description": "Added the **allowedCloudEndpoints** property to the [crossTenantAccessPolicy](https://docs.microsoft.com/en-us/graph/api/resources/crosstenantaccesspolicy?view=graph-rest-beta) resource.",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.DirectoryServices.json:10]: Suggestion to enhance the description.
```suggestion
          "Description": "Added the **allowedCloudEndpoints** property to the [crossTenantAccessPolicy](https://docs.microsoft.com/en-us/graph/api/resources/crosstenantaccesspolicy?view=graph-rest-beta) resource. Use this property to enable B2B collaboration across Azure clouds. For example, between tenants in Azure Global and Azure Government clouds, and Azure Global and Azure China clouds. ",
```

**Comment by jkdouglas** [comment]
PR that is unhiding this property in the public schema: https://msazure.visualstudio.com/One/_git/AD-AggregatorService-Workloads/pullrequest/6115061

**Pull request merged**